### PR TITLE
GPGM nerf

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

ROF from 0.15 to 0.2 seconds like it was before
damage_falloff_mult to 0.25 from 0.5 like it was before

## Why It's Good For The Game

GPMG is a cancerous weapon to deal with. The last buff made it almost as good for pushing than minigun, and that for free and roundstart. 

This gun is not thaat broke, but it encourages a way of playing that is atrocious to play against as a xenos, and frankly midly interesting to play as a marine (Requires zero skill?). 

## Changelog
:cl:
balance: GPGM ROF back to 0.2, damage_falloff_mult back to 0.25
/:cl:

### CODE REVIEW

**Comment by neohenry** [comment]
goodbye gpmg spam

**Comment by Walarks** [comment]
goodbye gpgm spam

**Comment by MiniMeatwad** [comment]
Thanks for fixing the GPMG.

**Comment by Swagile** [comment]
goodbye gpmg spam

**Comment by Dolth** [comment]
goodbye gpmg spam
hello smol smartgun

**Comment by EvilMelonTheSecond** [comment]
Just remove aim mode and make it a decent gun rather than sending it to Brazil with those stats that get outmatched by pistols.

**Comment by BraveMole** [comment]
> Just remove aim mode and make it a decent gun rather than sending it to Brazil with those stats that get outmatched by pistols.

I don't want this to be a decent gun. I much prefer playing against an unga with dual pistols, at least both the unga and me have fun

**Comment by EvilMelonTheSecond** [comment]
> 
> 
> > Just remove aim mode and make it a decent gun rather than sending it to Brazil with those stats that get outmatched by pistols.
> 
> I don't want this to be a decent gun. I much prefer playing against an unga with dual pistols, at least both the unga and me have fun.
> With this PR, GPMG still keeps its niche, and is a decent gun for beginner

Just remove it from the game then. With such terrible stats its a noob trap gun to give shrap to everybody. If the gun won't even have a stationary defense niche why even bother with this stat nerf to make it unusable?

**Comment by BraveMole** [comment]
> > > Just remove aim mode and make it a decent gun rather than sending it to Brazil with those stats that get outmatched by pistols.
> > 
> > 
> > I don't want this to be a decent gun. I much prefer playing against an unga with dual pistols, at least both the unga and me have fun.
> > With this PR, GPMG still keeps its niche, and is a decent gun for beginner
> 
> Just remove it from the game then. With such terrible stats its a noob trap gun to give shrap to everybody. If the gun won't even have a stationary defense niche why even bother with this stat nerf to make it unusable?

But i has a stationnary defence niche, it had it before the buff?

**Comment by Chaznoodles** [comment]
> > Just remove aim mode and make it a decent gun rather than sending it to Brazil with those stats that get outmatched by pistols.
> 
> I don't want this to be a decent gun.

hmm

**Comment by Pariah919** [comment]
why are you making this gun a "stationary defense gun" when you've literally made the game unplayable and instantly a loss if you're ever on the defense in most rounds?

You're nerfing this and making it into a noobtrap for no reason other than your belief of """fun"""
People find using guns like the gpmg fun, you're stating it isn't fun for the other player? You've stated that you don't want it to be viable even.

**Comment by TGCMbetcher** [comment]
Hello gmpg spam, 4 bullet a second aim mode

**Review by SplinterGP — CHANGES_REQUESTED** [review]
Give a better reasoning, the reasons you gave doens't even make sense compared to what you did of giving an even better aimmode to it, this just makes it worse on everything you said.

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -674,11 +674,11 @@
 	gun_skill_category = GUN_SKILL_HEAVY_WEAPONS
 	attachable_offset = list("muzzle_x" = 41, "muzzle_y" = 21,"rail_x" = 8, "rail_y" = 23, "under_x" = 25, "under_y" = 14, "stock_x" = 11, "stock_y" = 14)
 	actions_types = list(/datum/action/item_action/aim_mode)
-	aim_fire_delay = 0.2 SECONDS
+	aim_fire_delay = 0.05 SECONDS
```
Comment (BraveMole) [inline code/modules/projectiles/guns/rifles.dm:677]:
```suggestion
	aim_fire_delay = 0.2 SECONDS
```

**Comment by Pariah919** [comment]
Okay this now straight nerfing the gpmg for pretty flimsy reasons.

I don't really see your point in beliving that a staple normal kit weapon like the GPMG should be bad and that it needs to be nerfed when it isn't really a problem, and then state that its unfun to use for the player using it, and then state a uncommon gimmick loadout like akimbos is better overall (even though it probably is) that serves an entirely different role and purpose. 

This PR is wack

**Comment by SilencedMP5A5** [comment]
bad

**Comment by Tupinambis** [comment]
If the issue is that the GPMG is being used too much outside its suppressive fire role for pushing, why not just make it worse when you are moving around with it? Make it take longer to wield, make it feel heavier in general. 

It always felt to me like a weapon you should be setting up on an entrenched position with a bipod so you could even just make it recoil harshly/be unreliable and inaccurate when it's just being held.

**Comment by SplinterGP** [comment]
> 
> 
> If the issue is that the GPMG is being used too much outside its suppressive fire role for pushing, why not just make it worse when you are moving around with it? Make it take longer to wield, make it feel heavier in general.
> 
> It always felt to me like a weapon you should be setting up on an entrenched position with a bipod so you could even just make it recoil harshly/be unreliable and inaccurate when it's just being held.

it already is, using aimmode on it makes you almost unmovable

**Pull request closed without merging**